tests/networking/tests/networking_multus_links_negative.go

```diff
@@ -124,4 +124,43 @@ var _ = Describe("Networking custom namespace, ", func() {
 				globalparameters.TestCaseFailed)
 			Expect(err).ToNot(HaveOccurred())
 		})
+
+	It("custom deployment and daemonset 3 pods, 2 NADs, multiple Multus interfaces on deployment no "+
+		"connectivity via secondary interface[negative]", func() {
+
+		if len(multusInterfaces) < 2 {
```

Comment:
this is where I have some concerns, should not you let the test run and found that it's being skipped ?

Reply:
> this is where I have some concerns, should not you let the test run and found that it's being skipped ?

Yes. It's Ok. I still want to run all possible test cases on cluster that doesn't have 2 secondary interfaces